docs/api/timelapse.rst

```diff
@@ -56,21 +61,33 @@ Delete an unrendered timelapse
 
    Delete the unrendered timelapse ``name``.
 
+   Returns a :ref:`timelapse list <sec-api-timelapse-datamodel-list>` in the
+   response body.
+
    Requires the ``TIMELAPSE_DELETE`` permission.
 
+   :param unrendered: If provided and true, also include unrendered timelapses
+
 .. _sec-api-timelapse-saveconfig:
 
 Change current timelapse config
 ===============================
 
 .. http:post:: /api/timelapse
 
-   Save a new :ref:`timelapse configuration <sec-api-timelapse-datamodel-config>` to use for the next print.
+   Save a new :ref:`timelapse configuration <sec-api-timelapse-datamodel-config>` to use for the next print. Set the optional ``save`` flag in the
+   configuration to ``true`` in order to make the new configuration persist accross restarts.
 
    The configuration is expected as the request body.
 
+   Returns a :ref:`timelapse list <sec-api-timelapse-datamodel-list>` in the
+   response body.
+
    Requires the ``TIMELAPSE_ADMIN`` permission.
 
+   :param unrendered: If provided and true, also include unrendered timelapses
```

Comment:
```suggestion
   :param unrendered: If provided and true, also include unrendered timelapses in the response
```

Not as needed here since it is more obvious but if we're doing the other ones...